src/sql/parts/dashboard/containers/dashboardControlHostContainer.component.ts

```diff
@@ -53,6 +54,6 @@ export class DashboardControlHostContainer extends DashboardTab implements After
 	}
 
 	public refresh(): void {
-		// no op
+		this._hostContent.agentRefresh = true;
```

Comment:
I don't think the control host class should have methods specific to a particular child control.  The class is designed to host diverse experience "sub controls".  Like Agent or Profiler, for example.  The specific methods should be in the child classes.  At the host level the methods should be generic to any child, and should generally use interfaces which the child would implement.